### DISCUSSION TITLE
Grading Criteria for PA3

### DISCUSSION THREAD

**Discussion by rs0909** [ANLAB-KAIST/KENSv3#214] [Project Q&A 2. Transport Layer (TCP)] [answered]
Will there be a hidden case for project 2 and 3? Or can we get full credit if we pass all the test cases?

**Comment by Pusnow** [comment, marked as answer]
No, there are no hidden cases for PA 2 and 3.
If you pass all test cases, you will get full credit.
However, we will use `RANDOM_SEED` other than the default one.
Please test your implementation with many different `RANDOM_SEED`s.

**Reply by rs0909** [reply]
Got it! Thank you
